test/verifyIranianNationalId.spec.ts

```diff
@@ -1,26 +1,42 @@
 import { verifyIranianNationalId } from "../src";
 
+// https://www.searchline.ir/idcode
+
 it("Validation of Iranian National Number(code-e Melli)", () => {
-	expect(verifyIranianNationalId(123000000)).toBeFalsy();
-	expect(verifyIranianNationalId("0000000000")).toBeFalsy();
 	expect(verifyIranianNationalId("")).toBeUndefined();
 	expect(verifyIranianNationalId()).toBeUndefined();
-
+	expect(verifyIranianNationalId(0)).toBeUndefined();
+	expect(verifyIranianNationalId(12345)).toBeFalsy();
+	expect(verifyIranianNationalId("0")).toBeFalsy();
+	expect(verifyIranianNationalId("000000")).toBeFalsy();
+	expect(verifyIranianNationalId("12345")).toBeFalsy();
+	expect(verifyIranianNationalId(11537027)).not.toBeFalsy();
+	expect(verifyIranianNationalId(787833770)).not.toBeFalsy();
+	expect(verifyIranianNationalId(1583250689)).not.toBeFalsy();
+	expect(verifyIranianNationalId(12300000)).toBeFalsy();
+	expect(verifyIranianNationalId(123000000)).toBeFalsy();
+	expect(verifyIranianNationalId(1230000000)).toBeFalsy();
+	expect(verifyIranianNationalId("0000000000")).toBeFalsy();
+	expect(verifyIranianNationalId("1111111111")).toBeFalsy();
+	expect(verifyIranianNationalId("4444444444")).toBeFalsy();
+	expect(verifyIranianNationalId("9999999999")).toBeFalsy();
+	expect(verifyIranianNationalId("12300000")).toBeFalsy();
+	expect(verifyIranianNationalId("123000000")).toBeFalsy();
+	expect(verifyIranianNationalId("1230000000")).toBeFalsy();
 	expect(verifyIranianNationalId("0499370899")).not.toBeFalsy();
-	// https://www.yjc.ir/fa/news/6787095/%D8%B1%D9%86%D8%AF%D8%AA%D8%B1%DB%8C%D9%86-%D8%B4%D9%85%D8%A7%D8%B1%D9%87-%D9%85%D9%84%DB%8C-%DA%A9%D8%B4%D9%88%D8%B1-%D8%B1%D8%A7-%D8%A8%D8%B4%D9%86%D8%A7%D8%B3%DB%8C%D8%AF
```

Comment:
`1111111111` is a valid national id as I mentioned news before.

Reply:
It seems the national code of the subject has been wrongly assigned in the above link and for this reason he has encountered problems in all government departments. However, one way is to just verify the national code `1111111111`.